scripts/ci-collect-testruns.sh

```diff
@@ -4,8 +4,10 @@ SCRIPT_DIR=$( cd ${0%/*} && pwd -P )
 
 for pkg in ./packages/*/_testoutput.txt
 do
-    echo "processing $pkg"
-    junit=$(dirname "$pkg")/junit.xml
-    echo "dest $junit"
-    $SCRIPT_DIR/ava-etap-to-junit.mjs $pkg > $junit
+    if [[ $pkg != "./packages/*/_testoutput.txt" ]];    then
```

Comment:
lazy to handle non-expansion if there are no artifacts, in order to not fail the CI